generate/operation.go.tmpl

```diff
@@ -1,3 +1,5 @@
+const {{.Name}}Operation = `{{$.Body}}`
```

Comment:
May as well put a doc-comment on this? Probably just saying it's the query executed by `{{.Name}}`.